cstore_fdw.c

```diff
@@ -1136,6 +1143,9 @@ CStoreGetForeignPaths(PlannerInfo *root, RelOptInfo *baserel, Oid foreignTableId
 													   startupCost, totalCost,
 													   NIL,  /* no known ordering */
 													   NULL, /* not parameterized */
+#if PG_VERSION_NUM >= 90500
```

Comment:
Well, I think it is better to do this in the following pattern:
`If pg 9.5` 
function call with 9.5 API
`else`
function call with 9.4- API
`end if`

An example is on pg_shard as [here](https://github.com/citusdata/pg_shard/pull/158/files#diff-94066e53b61b35da3278923a1fd21124R99). I think that pg_shard's approach is more readable. What do you think?